samples/todo-app/database-api/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
+ARG ARCH=amd64
+FROM mcr.microsoft.com/dotnet/aspnet:7.0-cbl-mariner2.0-${ARCH} AS base
```

Comment:
Where are these Dockerfiles expected to be run from? If the Docker engine is running on an ARM machine, I think the correct architecture image should be picked up without needing to specify the architecture:
```
$ docker manifest inspect mcr.microsoft.com/dotnet/aspnet:7.0-cbl-mariner2.0
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1580,
         "digest": "sha256:98632866a382489cf38d1444411e7c631cf58d9e1efd4a3aed53dd0d7dea4354",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1579,
         "digest": "sha256:0e3a2ab6b9f7400926b422fbb35896b2f1e89cacb561cbf7da98406fc799471a",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
```

Reply:
These are sample apps we use for testing E2E, these dockerfiles are used to build images and pushed to ACR. deployment.yaml will use these images for deploying them to k8's cluster.

Reply:
This is taken care using docker buildx.

Reply:
Updated them as well.